quattor/types/system.pan

```diff
@@ -27,7 +27,7 @@ type structure_oldname = {
 include 'components/network/core-schema';
 
 # TODO: monitoring to be defined
-include 'monitoring/lemon/schema';
+include { if_exists('monitoring/lemon/schema') };
```

Comment:
My point was that the template may exist at a site but you may want not to include it in the context of some machines... For this you need a variable. Thus my proposal to have this variable allowing to specify a template name (namespace) and not to include anything if the variable is undef/null. If we go this way, I'd say that if the variable is defined, the template must exist: this makes troubleshooting easier than adding `if_exists()` but I am open make its existence optional...

I don't remember if a variable can be defined in a declaration template. If yes, we could do:
```
variable SYSTEM_MONITORING_SCHEMA ?= null;
include SYSTEM_MONITORING_SCHEMA;
```

If not, we should do:
```
include if ( is_defined(SYSTEM_MONITORING_SCHEMA ) ) SYSTEM_MONITORING_SCHEMA ;
```